roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -40,6 +40,16 @@
   when:
     - katello_repositories_pulp_version == "nightly"
 
+- name: 'Add Pulpcore repository'
+  yum_repository:
+    name: pulpcore-repository
+    description: Pulpcore nightly repository
+    baseurl: "https://koji.katello.org/releases/yum/katello-nightly/pulpcore/el{{ ansible_distribution_major_version }}/x86_64/"
```

Comment:
nightly should be a parameter is a parameter. Otherwise you can't use the 3.15 repos once they show up.

Reply:
@ekohl I'm now using the parameter in the baseurl and description